jxl/src/image.rs

```diff
@@ -268,6 +268,13 @@ impl<T: ImageDataType> Image<T> {
             image: self,
         }
     }
+
+    pub fn try_clone(&self) -> Result<Self> {
```

Comment:
do we also want to add `clone` if it can never fail?

Reply:
Ugh, this implementation is not what I wanted to write... now it can fail.